### DISCUSSION TITLE
Add multiple api spec support out of the box

### DISCUSSION THREAD

**Discussion by rexvel** [scalar/scalar#1379] [Ideas]
What do you think about supporting multiple API specifications to display on a single page? The files could be merged into one to fit the current scalar component's API (React, in my case).

For instance, if I have APIs X, Y, and Z, I'd like to display them on a single page, where each API would have its own separate section (similar to the pet, store, and user sections in the Petstore example).

Here's how the navigation might look. Additionally, such changes would require support for nested navigation for endpoints, in my opinion.

![image](https://github.com/scalar/scalar/assets/29538888/158e2ac5-0286-44c0-89a5-4168faec1141)

**Comment by marclave** [comment]
hey @rexvel ! we had another discussion created on this, and I really like this idea so im going to move it to an issue for more discussion :) 

we just need to figure out some things around search

**Reply by hanspagel** [reply]
Just for reference, the issue is here: https://github.com/scalar/scalar/issues/1851

**Reply by rexvel** [reply]
Thanks for the update, everyone. I apologize for the delayed response as I hadn't noticed the notification. Look forward to hearing more news from you!

**Comment by hanspagel** [comment]
works already
